src/analysis/normed_space/completion.lean

```diff
@@ -32,7 +32,7 @@ instance : normed_space 𝕜 (completion E) :=
   norm_smul_le := λ c x, induction_on x
     (is_closed_le (continuous_const_smul _).norm (continuous_const.mul continuous_norm)) $
     λ y, by simp only [← coe_smul, norm_coe, norm_smul],
-  .. completion.module 𝕜 E }
+  ..@completion.module 𝕜 E _ _ _ _ _ (normed_space.to_has_uniform_continuous_const_smul 𝕜 E) }
```

Comment:
Can you add the `normed_algebra` instance in this file too, now that it has an `algebra` instance to inherit from?